core/src/main/scala/org/http4s/Headers.scala

```diff
@@ -95,6 +96,14 @@ final class Headers private (headers: List[Header])
       this.toList.equals(otherheaders.toList)
     case _ => false
   }
+
+  /** Removes the `Content-Length`, `Content-Range`, `Trailer`, and
```

Comment:
Let's default to the scaladoc style.  I anticipate we'll adopt scalafmt, and that would be one less default to change.